tests/e2e/test_assign_entities.py

```diff
@@ -328,64 +344,24 @@ def test_command_assign_entities_no_reference_log(
     with no reference
     """
     collection_name = "ancient-woodland"
+    dataset = "ancient-woodland"
     collection = Collection(name=collection_name, directory=collection_dir)
     collection.load()
-
+    test_endpoint = "d779ad1c91c5a46e2d4ace4d5446d7d7f81df1ed058f882121070574697a5412"
     assign_entities(
         resource_file_paths=["mock_csv.csv"],
         collection=collection,
+        organisation=["government-organisation:D1342"],
         specification_dir=specification_dir,
         organisation_path=organisation_path,
         pipeline_dir=pipeline_dir,
+        dataset=dataset,
+        endpoints=test_endpoint,
     )
-
     assert "No reference" in caplog.text
     assert "mock_csv" in caplog.text
 
 
-def test_command_assign_entities_resource_not_processed(
```

Comment:
Just checking this test is no longer required.

Reply:
No we have isolated assign_entities now and we will provide the resource path, if not valid it will throw No Resource found which is included in other test case. This test case no longer is valid